apps/emqx_ft/src/emqx_ft_storage_exporter_fs.erl

```diff
@@ -49,12 +56,11 @@
 
 -type file_error() :: emqx_ft_storage_fs:file_error().
 
--opaque export() :: #{
+-type export_st() :: #{
```

Comment:
@keynslug Here I renamed all "export" naming to "export_st" — to avoid confusion with `Export = {ExportMod, ExportSt}` entities.